network-api/networkapi/wagtailpages/templates/wagtailpages/opportunity_page.html

```diff
@@ -35,5 +35,6 @@ <h1>{{ block.value }}</h1>
         {% endif %}
       {% endfor %}
     </div>
+    {% include "./static/ih_cta.html" %}
```

Comment:
we;re using the `static` dir for purely static page templates, so this should live in `fragments` instead.